MekHQ/src/mekhq/gui/view/UnitViewPanel.java

```diff
@@ -90,28 +85,28 @@ private void initComponents() {
 
 		setBackground(Color.WHITE);
 		
-		lblImage.setName("lblImage"); // NOI18N
+		lblImage.setName("lblImage"); //$NON-NLS-1$
 		lblImage.setBackground(Color.WHITE);
-		Image image = FluffImageHelper.getFluffImage(entity);
-		if(null == image) {
-			image = getImageFor(unit, lblImage);     
-		}
+
+        Image image = Optional.ofNullable(FluffImageHelper.getFluffImage(entity))
+                              .orElseGet(() -> Images.unit(iconPackage, unit, 120, 120));
         Icon icon = null;
-		if(null != image) {
-			if(image.getWidth(lblImage) > 200) {
-                image = image.getScaledInstance(200, -1, Image.SCALE_DEFAULT);               
+        if (null != image) {
+            if (image.getWidth(lblImage) > 120) {
+                image = image.getScaledInstance(120, -1, Image.SCALE_SMOOTH);
```

Comment:
200px was too wide IMHO, so I've made the "fluff" image narrower:  of course I'll revert it to 200 if you like it wider

<img width="522" alt="z" src="https://user-images.githubusercontent.com/471835/45562296-56e8fb00-b84a-11e8-9672-071d3e4785dd.png">